examples/README.md

```diff
@@ -0,0 +1,19 @@
+# Basic Nodejs example
+
+## How to use
+
+
+### SET ENV Variables
+``` 
+export PRIVATE_KEY=<paste your private key here> // A sample private key prefix with `0x`
+export FROM=<paste address belonging to private key here> // Your address 
+```
+
+Be careful with your private key, Use a  key your are comfortable with for development purpose.
+
+### Install and run:
+
+```bash
+cd examples
+npm run build
```

Comment:
Do we need this?

Reply:
IMO, No
but contributors finding difficulties in figuring out how to add functionality and test it.
so adding it but not added under examples, not in main readme

Reply:
LOL.